pom.xml

```diff
@@ -623,6 +623,11 @@
 			<artifactId>jvm-breakglass</artifactId>
 			<version>${jvm-breakglass.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.8.6</version>
```

Comment:
Why did you add this?

Reply:
Needed the JSON parser out of it since the `parameters` string is not in valid json. The parser helps convert the parameters to a valid json string before returning